.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,21 +11,17 @@ Happy contributing!
 
 ## What does this PR do?
 
-(Provide a description of what this PR does.)
+(Provide a description of what this PR does and why it's needed.)
```

Comment:
Can we follow PR template like this

## Objective

-Brief about the PR-

## Description

-Describe the change-

## How to Test

-Mention the steps to follow to verify the changes-

## Screenshots

-Add screenshots-

## Checklist

-[ ] Merged with latest branch
-[ ] Verified unit tests
-[ ] Code clean (Remove unnecessary codes)
-[ ] Attached all screenshots (Including FE)

I believe this will work for all the PRs like feature, bug fix. Please share your thoughts on this.

Reply:
Thanks for the suggestion, but I'd like to limit the scope of the change in this PR. If you feel strongly about your PR template, please submit an issue with info on why we should change to your suggested format.